db/seeds/dev/residents.js

```diff
@@ -34,10 +34,10 @@ exports.seed = function(knex) {
         interests: "Weightlifting, billiards, NY stock exchange",
         pets: "none",
         living_preferences: "I like to wake up early and stay active.",
-        semester: "Spring 2021",
+        semester: "Spring-2021",
```

Comment:
Nice fix, keeping the URL in mind! We can write a little function to slice out the '-' when we display on the UI.